docs/user/index.md

```diff
@@ -108,47 +110,46 @@ Atomist helps you work with GitHub in two ways:
 
 ### GitHub user authorization
 
-When the Atomist bot first arrives in a team, it will send a Direct Message
-to the authorizing user, requesting that they authorize Atomist
-to access GitHub on their behalf.
+When the Atomist bot first arrives in a team, it will send a direct
+message to the authorizing user, requesting that they authorize
+Atomist to access GitHub on their behalf.
 
 ![GitHub Authorization](img/github-auth.png)
 
-This same dialog will be shown to users anytime Atomist detects
-that an automation needs to access GitHub as that user.  Every user on the
+This same dialog will be shown to users anytime Atomist detects that
+an automation needs to access GitHub as that user.  Every user on the
 team must individually opt in.  Atomist will display this option each
-time an un-authorized user runs a Command
-that requires a GitHub authorization.
-Users can ask for their current GitHub authorization status by running:
+time an un-authorized user runs a command that requires a GitHub
+authorization.  Users can ask for their current GitHub authorization
+status by running:
 
 ```
 you> @atomist github
 ```
 
-Atomist will send a direct message to this user with their current Authorization
-status.
+Atomist will send a direct message to this user with their current
+GitHub authorization status.
 
 ### Organization webhooks
 
-GitHub organization members that have the [owner role][owners], are allowed
-to configure Organization-wide webhooks.  This is convenient
-because it only has to be configured once;
-however, you will require a user who has the `owner` role in
-your GitHub organization.
+GitHub organization members that have the [owner role][owners], are
+allowed to configure organization webhooks.  This is convenient
+because it only has to be configured once; however, you will require a
+user who has the `Owner` role in your GitHub organization.
 
 ```
 you> @atomist enroll org
 ```
 
-When you choose to enroll a GitHub Organization, you will most likely be
-prompted to authorize a new scope (Atomist only asks for new scopes when
-explicitly required).  The `admin:org_hook` is required when enrolling a new
-GitHub organization.
+When you choose to enroll a GitHub organization, you will most likely
+be prompted to authorize a new scope (Atomist only asks for new scopes
+when explicitly required).  The *admin:org_hook* is required when
+enrolling a new GitHub organization.
 
 ![GitHub Authorize Organization Webhook](img/authorize-org-hook.png)
 
-Since you might be a member of many GitHub organizations, Atomist may ask you to
-choose which organization you are enrolling.
+Since you might be a member of several GitHub organizations, Atomist
```

Comment:
really this should be "If you are a member of more than one GitHub organization, Atomist will ask you to choose which organization to enroll"